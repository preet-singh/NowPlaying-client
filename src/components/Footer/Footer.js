import React from 'react';

export default function Footer() {
  return(
    <a className="footer" href="http://www.tmdb.org" target="_blank" rel='noreferrer noopener'>Movie info from TMDB. Thanks!</a>
  )

}