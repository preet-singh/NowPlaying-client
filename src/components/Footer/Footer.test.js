import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { MemoryRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

describe.only('Footer testing', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <MemoryRouter>
            <Footer />
        </MemoryRouter>, 
    div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the footer', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
  
});