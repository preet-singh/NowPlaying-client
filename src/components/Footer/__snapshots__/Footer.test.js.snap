// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing renders the footer 1`] = `
<a
  className="footer"
  href="http://www.tmdb.org"
  target="_blank"
>
  Movie info from TMDB. Thanks!
</a>
`;
